package.json

```diff
@@ -35,6 +35,7 @@
     "@mui/material": "^5.12.2",
     "@mui/styled-engine-sc": "^5.12.0",
     "@mui/system": "^5.12.1",
+    "@mui/utils": "^5.14.18",
```

Comment:
What's this additional dependency? I can't find any documentation on it.

The npm repo makes it look like it may be private: https://www.npmjs.com/package/@mui/utils

Reply:
This is what MUI recommended for screen-reader-only text: https://mui.com/system/screen-readers/

Turns out the dep is already in there (check out package lock), so this isn't actually much extra for the bundle